Makefile.uk

```diff
@@ -0,0 +1,5 @@
+$(eval $(call addlib,apphelloworld))
+
+APPHELLOWORLD_SRCS-y += $(APPHELLOWORLD_BASE)/main.c
+APPHELLOWORLD_CFLAGS-$(CONFIG_OPTIMIZE_PGO_GENERATE) += -fprofile-generate
+#APPHELLOWORLD_CFLAGS-$(CONFIG_OPTIMIZE_PGO_USE) += -fprofile-use
```

Comment:
agree with Cezar, when an option is selected from the menuconfig, all the makefile lines associated with the option (CONFIG_option) will be executed, so the two lines can exist at the same time because only one will be executed at a time.